content/contribute/terms-and-conditions.md

```diff
@@ -7,3 +7,43 @@ linked:
 ---
 
 ## HCA Data Contributor Terms and Conditions
+
+### Preliminary terms and conditions for data use and sharing
+
+Version: 
+
+As soon as the official Human Cell Atlas terms and conditions for data contribution are finalised and reviewed by our legal authorities, you will be asked to agree to them. In that instance, we will delete all data you have already submitted to the Human Cell Atlas Data Coordination Platform.
+
+
+1. All data submitted to the Human Cell Atlas Data Coordination Platform will be made
+available for unrestricted public view, redistribution, and reuse. *
+
+2. All the data collaborators’ names and email addresses provided with the data will be
```

Comment:
You can actually number all of these items with "1. " and the markdown will render in increasing numbers. Avoids mistakes with missing/repeat numbers or when things get moved around.